src/state/LocalGameState.cpp

```diff
@@ -113,9 +113,9 @@ void LocalGameState::step_impl()
 	{
 		displayQueryPrompt(200,
 			TextManager::LBL_CONF_QUIT,
-			std::make_tuple(TextManager::LBL_YES, [&](){ switchState(new MainMenuState); }),
-			std::make_tuple(TextManager::LBL_NO,  [&](){ mMatch->unpause(); }),
-			std::make_tuple(TextManager::RP_SAVE, [&](){ mSaveReplay = true; imgui.resetSelection(); }));
+			std::make_tuple(TextManager::RP_SAVE, [&](){ switchState(new MainMenuState); }),
+			std::make_tuple(TextManager::LBL_OK,  [&](){ mMatch->unpause(); }),
+			std::make_tuple(TextManager::NET_STAY_ON_SERVER, [&](){ mSaveReplay = true; imgui.resetSelection(); }));
```

Comment:
I think wrong :)

Reply:
oh, yes, I did this just to test, shouldn't have been part of the commit